demos/README.md

```diff
@@ -41,6 +41,21 @@ kustomize build . >  kafka.yaml
 
 4. Copy the yaml file to `pixie/demos/kafka`.
 
+## Updating the `px-finagle` demo
+
+1. Clone `https://github.com/pixie-io/finagle-helloworld`
+
+2. (optional) Build the container images & update the individual yaml files.
```

Comment:
nit: "build the container image" since we will be using the same image for the client as well as server

Reply:
Updated.